### DISCUSSION TITLE
How add new user in Elsa Studio 3.1

### DISCUSSION THREAD

**Discussion by leostone68** [elsa-workflows/elsa-core#5749] [Design & tech]
How add new user in Elsa Studio 3.1?

**Comment by glime-ai** [comment]
To add a new user in Elsa Studio 3.1, you would typically follow the user management procedures provided by the application. However, based on the context provided, there is no direct documentation or guide that details the process of adding a new user specifically for Elsa Studio 3.1.

Elsa Studio, being a web-based application, might rely on the underlying ASP.NET Core Identity system for user management if it's set up that way. If Elsa Studio uses ASP.NET Core Identity (which is common for applications built on ASP.NET Core), you would usually add users through the Identity framework's user management tools, either programmatically or through a user interface provided by Elsa Studio if available.

Since the provided context does not include specific instructions for adding a user, you may need to refer to the general ASP.NET Core Identity documentation for guidance on managing users or look for user management features within Elsa Studio's interface.

If you are unable to find the option within the interface or if you are looking for a programmatic way to add users, you may need to extend the application by writing custom code that interacts with the Identity framework to create new users.

For a more precise answer, please provide additional details or clarify if there is a specific user management interface or documentation for Elsa Studio that you are referring to. If there is a user management section within Elsa Studio's documentation or interface, please follow the instructions provided there.

If you need further assistance or if there is a specific area of the documentation you would like me to look into, please let me know.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
